hack/istio/install-bookinfo-demo.sh

```diff
@@ -150,8 +160,11 @@ fi
 
 if [ "${DELETE_BOOKINFO}" == "true" ]; then
   echo "====== UNINSTALLING ANY EXISTING BOOKINFO DEMO ====="
-  if [[ "$CLIENT_EXE" = *"oc" ]]; then
-    $CLIENT_EXE delete network-attachment-definition istio-cni -n ${NAMESPACE}
+  if [[ "${IS_OPENSHIFT}" = "true" ]]; then
```

Comment:
double-equals, single brackets
```suggestion
  if [ "${IS_OPENSHIFT}" == "true" ]; then
```